Library/Formula/terraform.rb

```diff
@@ -145,14 +63,12 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
+      terraform_files = `go list ./... | grep -v /vendor/`.strip.gsub("\n", " ")
+      system "go test #{terraform_files}"
```

Comment:
@nicolai86 Like with the other arguments, you have to convert `terraform_files` from a space-delimited string (that is treated as a single argument if you pass it as such) to an array and then pass the entries of that array as individual arguments. Something like this should work:

```ruby
terraform_files = `go list ./... | grep -v /vendor/`.strip.split("\n")
system "go", "test", *terraform_files
system "gox", "-arch", arch, "-os", "darwin", "-output", "bin/terraform-{{.Dir}}", *terraform_files
```

A variant of the first line that does more of the work in Ruby would be:

```ruby
terraform_files = `go list ./...`.strip.split("\n").reject { |s| s.include?("/vendor/") }
```